src/main/java/com/redhat/devtools/intellij/tektoncd/utils/TreeHelper.java

```diff
@@ -75,6 +77,7 @@ public static Class retrieveNodeClassByKind(String kind) {
             case KIND_PIPELINES: return PipelinesNode.class;
             case KIND_RESOURCES: return ResourcesNode.class;
             case KIND_TASKS: return TasksNode.class;
+            case KIND_CONDITIONS: return ConditionsNode.class;
```

Comment:
Aren't we missing TaskRuns as well now ?

Reply:
Not now. This refresh is only called from the saveListener and it is only triggered from virtual files created by the plugin because of their hidden user data (so pipeline, task, clustetask, resources and condition). Taskruns are only created by starting a task. We need to add taskruns support when adding this https://github.com/redhat-developer/intellij-tekton/issues/89